src/SimpleS3.php

```diff
@@ -226,16 +220,17 @@ private function signRequest(
         );
         $signature = hash_hmac('sha256', $stringToSign, $signingKey);
 
-        $headers['authorization'] = "AWS4-HMAC-SHA256 Credential={$this->accessKeyId}/$scope,SignedHeaders=$headerNamesAsString,Signature=$signature";
+        $headers['authorization'] = "AWS4-HMAC-SHA256 Credential=$this->accessKeyId/$scope,SignedHeaders=$headerNamesAsString,Signature=$signature";
 
         return $headers;
     }
 
     private function getHostname(string $bucketName): string
     {
-        if ($this->region === 'us-east-1') return "$bucketName.s3.amazonaws.com";
-
-        return "$bucketName.s3-{$this->region}.amazonaws.com";
+        return match ($this->region) {
+            'us-east-1' => "$bucketName.s3.amazonaws.com",
+            default => "$bucketName.s3-$this->region.amazonaws.com",
+        };
```

Comment:
Same here, the match makes things more confusing to understand for most developers, I'd rather keep the `if`.

Reply:
You marked this one as resolved but didn't change the code?

Reply:
it is resolveed now I thought I resolved it when reverting the string interpolation 😢 my bad.